src/index.js

```diff
@@ -171,6 +173,18 @@ var VERSION = process.env.npm_package_version;
  *     </script>
  *   </body>
  * </html>
+ *
+ * @example
+ * <caption>Use your own translations</caption>
+ * braintree.dropin.create({
+ *   authorization: 'CLIENT_AUTHORIZATION',
+ *   container: '#dropin-container',
+ *   locale: {
+ *     payingWith: 'You are paying with {{paymentSource}}',
```

Comment:
This is slightly confusing to me. It looks like the merchant has to specify the strings they want for the customer rather than providing Drop-in with unsupported locales.

Say I want to support multiple locales, but I notice that Drop-in does not support `en_CA`. I might want to add my own `en_CA` but I still want Drop-in to load in my customer locales (`fr_CA`, `en_US`, etc).

Would it be better to have:
```
localizationOverride: {
    en_CA: {
      payingWith: '...'
    }
  }
```